### PR TITLE
Change Headlines

### PR DESCRIPTION
**Pull request by shaardie**
Apply changes to the headlines as discussed in the design meeting.
This should streamline the style.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -32,25 +32,25 @@ The project strives to create an application interface which lowers the barrier
...
 
 This project targets developers who want to build their own services based on
 the satellite data of the Copernicus program, but who do not want to work with
 huge amounts of scientific data directly. We will provide examples and
 libraries to quickly get you started without having to become an expert in
 satellite data analysis yourself.
 
-Publications
-------------
+In the News
```
Comment (lkiesow) [inline index.md]:
```suggestion
Media
```

File: index.md [hunk 2]
```diff
@@ -32,25 +32,25 @@ The project strives to create an application interface which lowers the barrier
...
 
 The project's core is an API which can be used to query the processed data.
 For this purpose we develop a cloud service which queries the freely
 accessible data of Sentinel-5P, aggregates it, stores it in a cache and makes
 it available.
 
 
-Target Audience
----------------
+Who is it for?
```
Comment (lkiesow) [inline index.md]: I'm no native speaker but this sounds very informal to me.
I would rather use
```suggestion
For whom is this?
```

Comment (gregorydlogan) [reply index.md]: @lkiesow IMO `Who is it for` is OK. I would reword to `Who is this (program/application/api/whatever) for`, but this change set as is seems fine to me.

Your suggestion sounds incredibly stuffy to me - I should be ringing the butler for a spot of tea should I use it :-). It's probably correct, but honestly I would have to look up the usage of whom to be sure. If that kind of formality is the goal then I would use your change, I think.

Comment (lkiesow) [reply index.md]: Okay, fair enough.
So, let's use

```suggestion
Who is the API for?
```

File: index.md (hunk 1 above)
Comment (shaardie) [reply index.md]: I actually like `In the News` more. It is hard to discuss about, since `Media` does describe the content of this part very well. I just have the feeling that `In the News` does sound fresher and more playful, but is still professional enough in my opinion. It sparks more joy :smile:

I hope this does make at least a little bit of sense.

Comment (lkiesow) [reply index.md]: But the only thing that one could remotely call “the news” in this category is the FLOSS Weekly episode. Everything else is no news story but talks at conferences.

Comment (tibroc) [reply index.md]: `Us talking about emissions API`

I want things that spark joy. I like joy.

Other options:

`Us talking`, `Us talking dirty (because emissions, you know ;)`,  `Oldies but Goldies`, `Say my name, say my name` (although I cut that part from the video), `because we couldn't afford to be on TED`, `look, our english is funny`

Comment (shaardie) [reply index.md]: @tibroc also likes `Media` more. So I changed it, since joy seems not the important anymore.
